### PR TITLE
update DG

### LINKED ISSUE #46: Update DG (Use case)

### PR DESCRIPTION
**Pull request by HollaG**

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -312,50 +312,127 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+MSS
+1. User chooses to add a new student.
+2. User enters the required details to create a student.
+3. TutorMate creates the student.
+4. The app shows the user the new student created.
+
+Use Case ends
+
+Extensions
+* 2a. An error parsing the user entered information was detected
```
Comment (elaineshijie) [inline docs/DeveloperGuide.md]: Would it be better if fewer implementation details are included (eg. User did not enter all the required details to create a student.) ?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -312,50 +312,127 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
+### Use case: Install the app [UC01]
 
-**Use case: Install the app**
 MSS 
-1. User execute the installation package
-2. The app ask for continue or cancel the installation
-3. User respond by indicating continue
-4. The app install the app
+1. User executes the installation package
+2. The app asks for continue or cancel the installation
```
Comment (pzl111) [inline docs/DeveloperGuide.md]: just a minor comment: is it better to say "asks to" rather than "asks for"

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -312,50 +312,127 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
...
 Extensions
-* 3a. User respond by indicating cancel
-  * 3a1. The app close the installation package
+* 3a. User responds by indicating cancel
+  * 3a1. The app closes the installation package
   * Use case ends
 
-* 4a. A error occur during installation
-  * 4a1. The app informs the user that the installation is unsuccessful and display reasons
+* 4a. An error occur during installation
```
Comment (richiehx) [inline docs/DeveloperGuide.md]: Should it be occurs instead of occur?

**Review by richiehx — COMMENTED** [review]
LGTM, some minor questions

**Comment by elaineshijie** [comment]
Fixes #46

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (HollaG) [reply docs/DeveloperGuide.md]: thanks for the suggestion

**Review by richiehx — APPROVED** [review]
LGTM!

**Pull request merged**